AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/participant.model.ts

```diff
@@ -23,6 +23,7 @@ export class ParticipantModel {
     is_interpretee?: boolean | undefined;
     user_role_name?: string | undefined;
     is_courtroom_account?: boolean;
+    addedDuringHearing?: boolean | undefined;
```

Comment:
You don't need a union type here - it's already marked as optional with the `?` so the union is unecessary

Reply:
Change implemented